contributed_definitions/nyaml/NXbias_spectroscopy.yaml

```diff
@@ -1,114 +1,92 @@
 category: base
 doc: |
-  Base classes definition for bias spectroscopy.
+  Base class definition for bias spectroscopy.
   
-  Bias sweeps while measuring arbitrary channels with I(V) curves.
+  The Scanning Tunneling Spectroscopy (STS), an extension of Scanning Tunneling Microscopy (STM),
+  is used to measure I(V) curves at each point of the surface. The dI/dV(V) can also be measured
+  with the help of numerical differentiation of the I(V) curve or from Lock-in amplifier.
 type: group
 NXbias_spectroscopy(NXobject):
-  channels:
-    doc: |
-      Select the channels to record.
-  reset_bias(NX_BOOLEAN):
-    doc: |
-      If chosen, the Bias voltage resets to its initial value (before the sweep initiation) at
-      the conclusion of the spectroscopy measurement. When this option is not selected, the Bias
-      voltage maintains the last value acquired during the sweep. This functionality proves
-      beneficial, especially when combining multiple sweep segments. As an illustration of an
-      automated measurement: turn off the z-Controller, commence spectroscopy sweep segments (
-      forward sweep only, without resetting the bias), restore the bias to its initial value,
-      and then turn on the z-Controller.
-  record_final_z(NX_BOOLEAN):
+  measurement_type:
     doc: |
-      Select whether to record the Z position during Z averaging time at the end of
-      the sweep (after Z control time) and store the average value in the header of
-      the file when saving. Using this option increases the sweep time by Z averaging
-      time.
-  lock_in_run(NX_BOOLEAN):
+      The measurement of the I(V) curve can come in two ways:
+        1. Constant Spacing: The bias voltage is swept from the start to the end with a constant
```

Comment:
Is not the bias voltage just a special dimension of scna_contol and as such this base class is a specialisation of scan_control?

Reply:
It could be considered as a special dimension of the NXscan_control, but it does not have a connection with the other dimension inside this base class, The reason is the measurement goes over a certain range of varying voltage only one single point. 

The connection comes when we use `NXscan_control` inside the application definition. If the experiments need such a `sweep voltage or bias_spectroscopy` method over a certain region or phase space then we can use `NXbias_scpectroscopy` inside the `NXscan_control` to explain the data and metadata of individual points of scan in the phase space or region.